### PR TITLE
enh(mc): automatic commit for network-cisco-ironport-snmp

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md [hunk 1]
```diff
@@ -2,48 +2,633 @@
...
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Cisco-Ironport-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                | Modèle de service                                   | Description                                               | Découverte |
+|:---------------------|:----------------------------------------------------|:----------------------------------------------------------|:----------:|
+| Mail-Usage           | Net-Cisco-Ironport-Mail-Usage-SNMP-custom           | Contrôle l'utilisation du mail                            |            |
+| Proxy-Usage          | Net-Cisco-Ironport-Proxy-Usage-SNMP-custom          | Contrôle l'utilisation du proxy                           |            |
+| Traffic-Generic-Id   | Net-Cisco-Ironport-Traffic-Generic-Id-SNMP-custom   | Contrôle le traffic réseau d'une interface réseau         |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md:37]:
```suggestion
| Traffic-Generic-Id   | Net-Cisco-Ironport-Traffic-Generic-Id-SNMP-custom   | Contrôle le trafic réseau d'une interface réseau         |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md [hunk 2]
```diff
@@ -2,48 +2,633 @@
...
+| Traffic-Generic-Name | Net-Cisco-Ironport-Traffic-Generic-Name-SNMP-custom | Contrôle le traffic réseau d'une interface réseau         |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md:38]:
```suggestion
| Traffic-Generic-Name | Net-Cisco-Ironport-Traffic-Generic-Name-SNMP-custom | Contrôle le trafic réseau d'une interface réseau         |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md [hunk 3]
```diff
@@ -2,48 +2,633 @@
...
+| Traffic-Global       | Net-Cisco-Ironport-Traffic-Global-SNMP-custom       | Contrôle le traffic réseau de plusieurs interfaces réseau | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md:39]:
```suggestion
| Traffic-Global       | Net-Cisco-Ironport-Traffic-Global-SNMP-custom       | Contrôle le trafic réseau de plusieurs interfaces réseau | X          |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md [hunk 4]
```diff
@@ -2,48 +2,632 @@
...
+| Memory          | Net-Cisco-Ironport-Memory-SNMP-custom          | Check memory usage                                   |
+
+> The services listed above are created automatically when the **Net-Cisco-Ironport-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                                    | Service Description                          | Discovery  |
+|:---------------------|:----------------------------------------------------|:---------------------------------------------|:----------:|
+| Mail-Usage           | Net-Cisco-Ironport-Mail-Usage-SNMP-custom           | Check email security usage                   |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md:34]: security?

File: pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md [hunk 5]
```diff
@@ -2,48 +2,632 @@
...
+| Proxy-Usage          | Net-Cisco-Ironport-Proxy-Usage-SNMP-custom          | Check proxy usage                            |            |
+| Traffic-Generic-Id   | Net-Cisco-Ironport-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface        |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md:36]:
```suggestion
| Traffic-Generic-Id   | Net-Cisco-Ironport-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface        |            |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md [hunk 6]
```diff
@@ -2,48 +2,632 @@
...
+| Traffic-Generic-Name | Net-Cisco-Ironport-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface        |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md:37]:
```suggestion
| Traffic-Generic-Name | Net-Cisco-Ironport-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface        |            |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md (hunk 4 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-cisco-ironport-snmp.md:34]:
```suggestion
| Mail-Usage           | Net-Cisco-Ironport-Mail-Usage-SNMP-custom           | Check email usage                   |            |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**